ld: duplicate symbol _saved_plain in NETNTLMv2_fmt.o and NETLMv2_fmt.o
collect2: ld returned 1 exit status


--- src/NETNTLMv2_fmt.c
+++ src/NETNTLMv2_fmt.c
@@ -77,9 +77,9 @@
   {NULL}
 };
 
-uchar saved_plain[PLAINTEXT_LENGTH + 1];
-uchar challenge[SALT_SIZE_MAX + 1];
-uchar output[BINARY_SIZE + 1];
+static uchar saved_plain[PLAINTEXT_LENGTH + 1];
+static uchar challenge[SALT_SIZE_MAX + 1];
+static uchar output[BINARY_SIZE + 1];
 
 extern void E_md4hash(uchar *passwd, uchar *p16);
 extern void hmac_md5_init_limK_to_64(const unsigned char*, int, HMACMD5Context*);
--- src/NETLMv2_fmt.c
+++ src/NETLMv2_fmt.c
@@ -74,9 +74,9 @@
   {NULL}
 };
 
-uchar saved_plain[PLAINTEXT_LENGTH + 1];
-uchar challenge[SALT_SIZE + 1];
-uchar output[BINARY_SIZE + 1];
+static uchar saved_plain[PLAINTEXT_LENGTH + 1];
+static uchar challenge[SALT_SIZE + 1];
+static uchar output[BINARY_SIZE + 1];
 
 extern void E_md4hash(uchar *passwd, uchar *p16);
 extern void hmac_md5_init_limK_to_64(const unsigned char*, int, HMACMD5Context*);
